.changeset/shiny-turkeys-kiss.md

```
@@ -0,0 +1,5 @@
+---
+'@papra/app': patch
+---
+
+Document sort order fallback to the importation date when the sort order is set to other sort orders.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)